src/main/java/Duke.java

```diff
@@ -1,10 +1,223 @@
+
+        switch (taskType) {
+            case TASK_TYPE_TODO:
+                Todo todo = new Todo(taskDescription);
+                taskArrayList.add(todo);
+                todo.printAddTaskMessage();
+                break;
+            case TASK_TYPE_DEADLINE:
+                String deadlineTask = getTaskFromTaskDescription(taskDescription, DEADLINE_DATE_INDICATOR);
+                String deadlineDate = getDateFromTaskDescription(taskDescription, DEADLINE_DATE_INDICATOR);
+                Deadline deadline = new Deadline(deadlineTask, deadlineDate);
+                taskArrayList.add(deadline);
+                deadline.printAddTaskMessage();
+                break;
+            case TASK_TYPE_EVENT:
+                String eventTask = getTaskFromTaskDescription(taskDescription, EVENT_DATE_INDICATOR);
+                String eventDate = getDateFromTaskDescription(taskDescription, EVENT_DATE_INDICATOR);
+                Event event = new Event(eventTask, eventDate);
+                taskArrayList.add(event);
+                event.printAddTaskMessage();
+                break;
+        }
+        printNumberOfTask();
+    }
+
+    public static void markTaskAsDone(String userInput, ArrayList<Task> taskArrayList) {
+        String[] userInputArray = userInput.split(USER_INPUT_SPLITTER);
+        int doneTaskNumber = Integer.parseInt(userInputArray[1]);
+
+        // We take doneTaskNumber and subtract 1 from it
+        // since ArrayList's index starts from 1
+        doneTaskNumber = doneTaskNumber - 1;
+        Task taskDone = taskArrayList.get(doneTaskNumber);
+        taskDone.setTaskAsDone();
+        taskDone.printMarkTaskAsDone();
+    }
+
+    public static void getAllTask(ArrayList<Task> taskArrayList) {
+        System.out.println("Here are the tasks in your list:");
+        int taskNumber;
+        for (int i = 0; i < taskArrayList.size(); i ++) {
+            Task currentTask = taskArrayList.get(i);
+            taskNumber = i + 1;
+            System.out.println(taskNumber + "." + currentTask.toString());
+        }
+        System.out.println();
+    }
+
+    public static void deleteTaskMessage(Task taskToDelete) {
+        System.out.println("Noted. I've removed this task:\n" +
+                "  " + taskToDelete.toString());
+    }
+
+    public static void deleteTask(String userInput, ArrayList<Task> taskArrayList) {
+        String[] userInputArray = userInput.split(USER_INPUT_SPLITTER);
+        int deleteTaskNumber = Integer.parseInt(userInputArray[1]);
+        int indexOfTaskToDelete = deleteTaskNumber - 1;
+
+        Task taskToDelete = taskArrayList.get(indexOfTaskToDelete);
+        taskArrayList.remove(indexOfTaskToDelete);
+        deleteTaskMessage(taskToDelete);
+        printNumberOfTask();
+    }
+
+    public static void printInvalidTaskNumberProvided() {
+        System.out.println("Task number provided does not exist!\n");
+    }
+
     public static void main(String[] args) {
```

Comment:
This method is too long ([>30 LoC](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)).  

To shorten the code, how about abstracting some blocks of code out and ensuring [SLAP](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard)?